src/main/java/VASSAL/build/module/GameRefresher.java

```diff
@@ -320,21 +325,24 @@ public void windowClosing(WindowEvent we) {
         @Override
         public void actionPerformed(ActionEvent e) {
```

Comment:
Since you're already making changes right here---these `ActionListeners` could all become lambdas. E.g.,
```
runButton.addActionListener(e -> run());
```